pom.xml

```diff
@@ -99,7 +99,7 @@ THE SOFTWARE.
     <changelog.url>https://jenkins.io/changelog</changelog.url>
 
     <!-- Bundled Remoting version -->
-    <remoting.version>4.7</remoting.version>
+    <remoting.version>4.8-rc2925.e195d94d1423</remoting.version>
```

Comment:
We will need to list all notable changes from https://github.com/jenkinsci/remoting/releases/tag/remoting-4.8 in the changelog proposal, not just your change

Reply:
I got it.

Reply:
I'll change the changelog proposal and PR title after the new remoting release including https://github.com/jenkinsci/remoting/pull/452. Sorry for the bad timing additional PR...